projects/laji/src/app/+taxonomy/taxon/info-card/taxon-identification/taxon-identification.component.ts

```diff
@@ -49,12 +57,93 @@ export class TaxonIdentificationComponent implements OnChanges, AfterViewInit, O
     viewChange: this.infiniteScrollStatusCheck$.pipe(
       filter(() => this.loadMoreElem && this.isWithinXPixelsOfViewport(this.loadMoreElem.nativeElement, INFINITE_SCROLL_DISTANCE)),
       map(() => ({
-          start: 0,
-          end: this.children.length
-        }))
+        start: 0,
+        end: this.children.length
+      }))
     )
   };
 
+  private parseTaxonDescriptions(child: Taxonomy) {
+    if (!child.descriptions || child.descriptions.length < 1) { return undefined; }
+
+    const descriptions = child.descriptions;
+    const taxonDescriptions = {};
+
+    const requestedVariables = {
```

Comment:
Might make it easier to read the class if long dictionaries like this are moved outside of the class (can be in the same file even). They are more 'data-y' than 'logic-y'.